pages/common/axel.md

```diff
@@ -0,0 +1,24 @@
+# axel
+
+> Download accelerator
+> Supports HTTP, HTTPS, and FTP
+
+- Download a URL to a file
+
+`axel {{url}}`
+
+- Download and rename
+
+`axel {{url}} -o {{filename}}`
+
+- Download with multiple connections
+
+`axel -n {{20}} {{url}}`
```

Comment:
is 20 the amount of connections? Then it should be something like number_of_connections